README.md

```diff
@@ -36,10 +45,14 @@ module ([`/example`](examples/)),
 ```
 
 ## Handling Blockstack sessions
```

Comment:
```
Blockstack sessions are stored in the `SessionStore`. The default implementation of `SessionStore` uses 
the default shared preferences. If an apps needs two `BlockstackSession` objects then they should 
use the same session store or synchronize the state regularly.  
```

-  Not clear, is it `SessionStore` or `BlockstackSession`? If both are needed, explain the relationship between the two.
- Do we have a synchronize method we can point readers to?